frontend/awx/access/credentials/components/CredentialMultilineInput.tsx

```diff
@@ -26,16 +32,24 @@ export function CredentialMultilineInput({
   accumulatedPluginValues: CredentialPluginsInputSource[];
   setAccumulatedPluginValues?: (values: CredentialPluginsInputSource[]) => void;
   setPluginsToDelete?: React.Dispatch<React.SetStateAction<string[]>>;
+  isSecret?: boolean;
+  pluginsToDelete?: string[];
+  initialValues?: initialValues;
 }) {
   const { t } = useTranslation();
+  const [shouldHideField, setShouldHideField] = useState(isSecret);
+  const { getValues } = useFormContext();
+  const watchedFieldValue = getValues();
   const { setValue, clearErrors } = useFormContext();
   const onClear = () => {
     setValue(field.id, '', { shouldDirty: false });
     clearErrors(field.id);
     setAccumulatedPluginValues?.(
       accumulatedPluginValues.filter((cp) => cp.input_field_name !== field.id)
     );
-    setPluginsToDelete?.((prev: string[]) => [...prev, field.id]);
+    watchedFieldValue?.[field.id]?.includes('Value is managed by')
```

Comment:
This string is selected to be translated...we might re-evaluate this logic.

Reply:
@nixocio good point. @skochay330 i think we can check `accumulatedPluginValues` to see if the credential was a plugin, and then mark it for deletion. I would do this prior to filtering it out of the `accumulatedPluginValues` array:

```tsx
// if field value was previously managed by a credential plugin, mark it for deletion
    if (accumulatedPluginValues.some((cp) => cp.input_field_name === field.id)) {
      setPluginsToDelete?.((prev: string[]) => [...prev, field.id]);
    }
```